### PR TITLE
(ru) Fix cursor images at "CSS/cursor" page

### LINKED ISSUE #3925: Issue with "Курсор": (missed example images)
MDN URL: https://developer.mozilla.org/ru/docs/Web/CSS/cursor

#### What information was incorrect, unhelpful, or incomplete?
Missed examples images
image

#### Specific section or headline?
Изменение размера, прокручивание (скроллинг)

#### What did you expect to see?
Example images

#### Did you test this? If so, how?
image

<details>
<summary>MDN Content page report details</summary>

* Folder: `ru/web/css/cursor`
* MDN URL: https://developer.mozilla.org/ru/docs/Web/CSS/cursor
* GitHub URL: https://github.com/mdn/translated-content/blob/main/files/ru/web/css/cursor/index.html
* Last commit: https://github.com/mdn/translated-content/commit/509c3fd223e0370e1ae2adc937c67b336d85df87
* Document last modified: 2022-01-30T16:28:58.000Z

</details>

### PR DESCRIPTION
**Pull request by cw118**
- Fixes image sources for CSS cursor (e.g. from `/@api/deki/files/3438/=default.gif` to `default.gif`)
- Fixes/adds corresponding cursor styles for `grab` and `grabbing` (e.g. `style="cursor: grab;"`)
- Adds `browser-compat` into front matter

Fixes #3925

### CODE REVIEW

File: files/ru/web/css/cursor/index.html [hunk 1]
```diff
@@ -55,7 +56,7 @@ <h3 id="Значения">Значения</h3>
    </tr>
    <tr style="cursor: default;">
     <td><code>default</code></td>
-    <td><img alt="default.gif" src="/@api/deki/files/3438/=default.gif" style="height: 26px; width: 26px;"></td>
+    <td><img alt="default.gif" src="default.gif"></td>
```
Comment (sashasushko) [inline files/ru/web/css/cursor/index.html:59]: There is no image - https://developer.mozilla.org/ru/docs/Web/CSS/cursor/default.gif

Comment (sashasushko) [reply files/ru/web/css/cursor/index.html:59]: Only at English version - https://developer.mozilla.org/en-US/docs/Web/CSS/cursor/default.gif

Comment (sashasushko) [reply files/ru/web/css/cursor/index.html:59]: So, we can use paths like that `/en-US/docs/Web/CSS/cursor/default.gif`

**Review by sashasushko — CHANGES_REQUESTED** [review]

**Comment by cw118** [comment]
Hi @sashasushko, I believe since this doesn't involve live samples, just using the file name will suffice. For example, the same strategy worked with #3331 and many other PRs of mine.

**Review by sashasushko — APPROVED** [review]
@cw118 okay, let's try.

**Comment by sashasushko** [comment]
It takes time to deploy. Maybe hours

**Comment by cw118** [comment]
Thank you for trying this out! I'll check the page a bit later (once the changes have been deployed), and if the images don't display properly, I'd be happy to correct the `src`s in another PR.

**Pull request merged**